inc/alert.class.php

```diff
@@ -465,9 +465,16 @@ static function displayAlerts($params = []) {
             echo "</div>";
          }
       }
+
+      $hidden_params = [
+         'show_hidden_alerts'          => true,
+         'show_only_login_alerts'      => false,
+         'show_only_helpdesk_alerts'   => true,
```

Comment:
```suggestion
         'show_only_helpdesk_alerts'   => $p['show_only_helpdesk_alerts'],
```

Reply:
Ok i see, 

I didn't understand why the value of ```$p['show_only_helpdesk_alerts']``` was always false 

that's why I've forced the value to true

this check is wrong
```php
   static function displayOnCentral() {
      echo "<tr><th colspan='2'>";
      self::displayAlerts(['show_only_helpdesk_alerts' => Session::getCurrentInterface() == 'central']);
      echo "</th></tr>";
   }
```

We need to check if central is displayed

Fixed